### PR TITLE
Vert.x 3.9.5

### PR DESCRIPTION
**Pull request by jponge**
Bump to 3.9.5

### CODE REVIEW

File: Formula/vert.x.rb [hunk 1]
```diff
@@ -1,15 +1,10 @@
...
-  sha256 "ea81f1c64333d2372cc9a628c7a7c3491b4ba5b059558fb84137c5b2bf4ecc9e"
+  url "https://bintray.com/vertx/downloads/download_file?file_path=vert.x-3.9.5-full.zip"
+  sha256 "fd8eb693f34f526280b7b2d7e6032212dbd56665b023a855ae69cb01a464de75"
   license any_of: ["EPL-2.0", "Apache-2.0"]
 
-  livecheck do
-    url "https://vertx.io/download/"
-    regex(/href=.*?vert\.x[._-]v?(\d+(?:\.\d+)+)-full\.t/i)
-  end
-
```
Comment (Rylan12) [inline Formula/vert.x.rb:12]: Why is this removed?

Comment (Rylan12) [reply Formula/vert.x.rb:12]: Oh, I think that the url just needs to be updated to https://bintray.com/vertx/downloads/distribution/3.9.4/view

(Per https://vertx.io/download/)

Comment (Rylan12) [reply Formula/vert.x.rb:12]: Although it looks like version 4.0.0 has been released. Can we update to that version?

Comment (jponge) [reply Formula/vert.x.rb:12]: Vert.x 4 is managed as a separate TAP.

Comment (jponge) [reply Formula/vert.x.rb:12]: > Oh, I think that the url just needs to be updated to https://bintray.com/vertx/downloads/distribution/3.9.4/view
> 
> 
> 
> (Per https://vertx.io/download/)

Yes the new website broke this, but I'm not sure we need this anyway.

Comment (bayandin) [reply Formula/vert.x.rb:12]: > Vert.x 4 is managed as a separate TAP.

homebrew-core formula should always point to the latest stable version, if 4.0 is so, then it should be upgraded

Comment (jponge) [reply Formula/vert.x.rb:12]: 3.9.5 is the latest stable version of the 3.9.5 branch.

4.0.0 is maintained in a tap, and we are decreasing the emphasis on that type of download so 3.9.5 should really be what's in Brew.

Comment (vietj) [reply Formula/vert.x.rb:12]: 4.x has a _vertx4_ formula in order to avoid people overwrite their 3.x with a 4.x installation

Comment (SMillerDev) [reply Formula/vert.x.rb:12]: brew should be shipping the latest version of any branch. That's what it's an evergreen package manager for.

Comment (jponge) [reply Formula/vert.x.rb:12]: Sure we get it, but as the people behind Vert.x we think it's better for users if the `vert.x` formula sticks to 3.9.x LTS and we maintain 4.0 as a separate tap.

Comment (SMillerDev) [reply Formula/vert.x.rb:12]: That's unfortunately not up to you, the policy for homebrew means that we will always ship the latest version of software. As long as Vert.x is open source we can and will continue to do that.

---
Side note: You might want to check what python did with version 2 and 3 to see why the current course of action might not have the result you want

Comment (jponge) [reply Formula/vert.x.rb:12]: @vietj What should we do here?

Comment (vietj) [reply Formula/vert.x.rb:12]: I think we might consider dropping supporting this formula because the Homebrew policy does not take in account specific needs from the community. Instead we would use the TAP we created with a vertx3 formula and deprecate the vert.x formula.

Comment (SMillerDev) [reply Formula/vert.x.rb:12]: That has the same issue as not updating, vertx being open source still means we can and will update the formula.  See https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap#official-vendor-taps

**Comment by Rylan12** [comment]
Closing in favor of #66946

File: Formula/vert.x.rb (hunk 1 above)
Comment (vietj) [reply Formula/vert.x.rb:12]: as you wish, we won't make a big deal of that
